samples/geotag.cpp

```diff
@@ -262,8 +282,18 @@ time_t Position::deltaMax_ = 60 ;
 class UserData
 {
 public:
-    UserData(Options& options) : indent(0),count(0),nTrkpt(0),bTime(false),bEle(false),options_(options) {};
-    virtual ~UserData() {} ;
+    UserData(Options& options):
```

Comment:
[suggestion] This is used as a POD struct => make it one (it hence doesn't need a constructor, as it should be value initialized, or `memset`'d)

Reply:
Sure, it's not really a problem anyway.